packages/console/src/pages/TenantSettings/Subscription/index.tsx

```diff
@@ -18,14 +24,33 @@ function Subscription() {
     currentSubscription,
     onCurrentSubscriptionUpdated,
   } = useContext(SubscriptionDataContext);
+  const { currentTenantId } = useContext(TenantsContext);
 
   const reservedPlans = pickupFeaturedPlans(subscriptionPlans);
   const reservedSkus = pickupFeaturedLogtoSkus(logtoSkus);
 
+  const { data: periodicUsage } = useSWR(
+    isCloud &&
+      isDevFeaturesEnabled &&
+      `/api/tenants/${currentTenantId}/subscription/periodic-usage`,
+    async () =>
+      cloudApi.get(`/api/tenants/:tenantId/subscription/periodic-usage`, {
+        params: { tenantId: currentTenantId },
+      })
```

Comment:
This dev feature enabled guard seems to be a little problem right now. I think it deserves a short discussion

Reply:
If we keep it, then the `<CurrentPlan />` component will be problematic, but if we remove it, we need to have the mindset that we cannot deploy the Cloud master branch to production, as it introduces a breaking change.

Reply:
Fixed the compatibility, use `currentTenant`'s usage as fallback to avoid breaking change.